src/Core/Services/Implementations/CollectionService.cs

```diff
@@ -41,14 +41,24 @@ public class CollectionService : ICollectionService
     }
 
     public async Task SaveAsync(Collection collection, IEnumerable<CollectionAccessSelection> groups = null,
-        IEnumerable<CollectionAccessSelection> users = null, Guid? assignUserId = null)
+        IEnumerable<CollectionAccessSelection> users = null)
     {
         var org = await _organizationRepository.GetByIdAsync(collection.OrganizationId);
         if (org == null)
         {
             throw new BadRequestException("Organization not found");
         }
 
+        var groupsList = groups?.ToList();
+        var usersList = users?.ToList();
+        var groupHasManageAccess = groupsList?.Any(g => g.Manage) ?? false;
+        var userHasManageAccess = usersList?.Any(u => u.Manage) ?? false;
+        if (!groupHasManageAccess && !userHasManageAccess)
+        {
+            throw new BadRequestException(
+                "At least one User or Group must have Manage access to the newly created collection.");
```

Comment:
issue: The `SaveAsync()` method isn't always creating a new collection so we should drop that part of the exception message.

Side-note, we ended up using slightly different language for our error messages. I used the following on the client to match more closely with the access dialog language:
`At least one member or group must have can manage permission.`

Might be worthwhile to reach out to product to determine what language they want because I think yours could work just as well.